astropy/io/votable/converters.py

```diff
@@ -493,7 +493,7 @@ def __init__(self, field, base, arraysize, config=None, pos=None):
         Array.__init__(self, field, config)
 
         self._base = base
-        self.default = np.array([], dtype=self._base.format)
+        self.default = np.empty(0, dtype=self._base.format)
```

Comment:
Looks like this is the only case where `[]` is used with `self._base.format`. I think we should wait till Numpy wheel picks up that patch (and hence fails our numpy-dev job on `master`) and then make sure the numpy-dev job here passes with these changes. Otherwise, LGTM on the `io.votable` side. Thank you!